backends/advanced/src/advanced_omi_backend/services/audio_stream/producer.py

```
@@ -187,14 +181,22 @@ async def get_session(self, session_id: str) -> dict:
+                    v.decode() if isinstance(v, bytes) else v
+                )
+                for k, v in session_data.items()
+            }
+            if session_data
+            else {}
+        )
 
     async def update_session_job_ids(
         self,
         session_id: str,
         speech_detection_job_id: str = None,
```

Ruff (0.15.1)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)